### PR TITLE
[Liniting] Updated to biome v1.9

### PR DESCRIPTION
**Pull request by KenAJoh**
### Description

Ignored rules: 
- useSemanticElements: We have lots of cases where the native elements don't stack up to custom implementations. Some examples for changes they want: role combobox -> `<select>`, role status -> `<output>`. Best to ignore for now, but wold be a good idea in the future to do a collective walgrough and check if we can change some things.

Remaining changes are updates recommended by Biome, like not needed React.Fragments, duplicate border and border-color and unknown media queries

@HalvorHaugan Could you test the page https://aksel.nav.no/god-praksis/artikler/slik-gjor-du-en-lavterskel-brukertest-av-tekst with some different image sizes while toggling translateZ and image-rendering CSS on the `` element? I notice not difference in image quality on Mac, but might be a chrome on windows only thing. 

Screenshot 2024-10-03 at 09 42 21

### Component Checklist 📝

- [x] JSDoc
- [x] Examples
- [x] Documentation
- [x] Storybook
- [x] Style mappings (`@navikt/core/css/config/_mappings.js`)
- [x] Component tokens (`@navikt/core/css/tokens.json`)
- [x] CSS class deprecations (`@navikt/aksel-stylelint/src/deprecations.ts`)
- [x] Exports (`@navikt/core/react/src/index.ts` and `@navikt/core/react/package.json`)
- [x] New component? CSS import (`@navikt/core/css/index.css`)
- [x] Breaking change? Update migration guide. Consider codemod.
- [x] Changeset (Format: `<Component>: <gitmoji?> <Text>.` E.g. "Button: :sparkles: Add feature xyz.")

### CODE REVIEW

**Comment by HalvorHaugan** [comment]
> Could you test the page https://aksel.nav.no/god-praksis/artikler/slik-gjor-du-en-lavterskel-brukertest-av-tekst with some different image sizes while toggling translateZ and image-rendering CSS on the  element? I notice not difference in image quality on Mac, but might be a chrome on windows only thing.

Did not notice any difference on Windows, tested Chrome and FF. But `image-rendering` had an effect in Safari 18 (tested in BrowserStack). I think the images looks worse with this CSS property. There are probably some edge cases where some images would look better ([examples](https://css-tricks.com/almanac/properties/i/image-rendering/)), but afaics it's a bad idea to apply it to all images in general.

**Comment by KenAJoh** [comment]
@HalvorHaugan Thank you! No issue just removing them then 🚀

File: @navikt/core/react/src/date/datepicker/datepicker.stories.tsx [hunk 1]
```diff
@@ -73,40 +73,37 @@ export const Default: StoryObj<DefaultStoryProps> = {
...
+                ) : (
+                  <>
+                    <DatePicker.Input
+                      label="Velg dato"
+                      size={props?.size}
+                      {...singleCtx.inputProps}
+                    />
+                  </>
+                )}
+              </>
```
Comment (HalvorHaugan) [inline @navikt/core/react/src/date/datepicker/datepicker.stories.tsx:99]: Can even remove the remaining fragments

File: aksel.nav.no/website/components/styles/global.css [hunk 2]
```diff
@@ -38,16 +38,6 @@ html:has(.globalstyles) {
...
-/* Improves rendering of images for chrome on Windows */
-/* stylelint-disable-next-line media-feature-name-no-vendor-prefix */
-@media screen and (-webkit-min-device-pixel-ratio: 0) and (min-resolution: 0.001dpcm) {
-  .globalstyles img {
-    transform: translateZ(0);
-    /* stylelint-disable-next-line value-no-vendor-prefix */
-    image-rendering: -webkit-optimize-contrast !important;
-  }
-}
-
```
Comment (HalvorHaugan) [inline aksel.nav.no/website/components/styles/global.css:50]: Then we can also remove the stuff bellow

**Comment by HalvorHaugan** [comment]
> useSemanticElements: (...) Best to ignore for now, but wold be a good idea in the future to do a collective walgrough and check if we can change some things.

Maybe create an issue for this, so that we don't forget?

File: biome.json [hunk 3]
```diff
@@ -25,7 +25,8 @@
         "useMediaCaption": "off",
         "useKeyWithClickEvents": "off",
         "noNoninteractiveTabindex": "off",
-        "useAriaPropsForRole": "off"
+        "useAriaPropsForRole": "off",
```
Comment (HalvorHaugan) [inline biome.json]: Why turn off `useAriaPropsForRole`?

**Comment by KenAJoh** [comment]
> Maybe create an issue for this, so that we don't forget?

https://github.com/navikt/team-aksel/issues/646

File: biome.json (hunk 3 above)
Comment (KenAJoh) [reply biome.json]: Removed it now 👍

**Review by HalvorHaugan — APPROVED** [review]

**Pull request merged**